internal/home/service.go

```diff
@@ -531,14 +543,22 @@ status() {
 // guarantees that it will actually be the required directory.
 //
 // See https://github.com/AdguardTeam/AdGuardHome/issues/2614.
+//
+// Improvement to FreeBSD rc script to allow it be controlled from rc.conf
```

Comment:
The informative part of a comment is usually *above* todos and starts with the name of the identifier it is documenting:

> freeBSDScript is an improved version […]